README.md

```diff
@@ -38,24 +38,38 @@ During this recording, we:
 
 [![demo](https://asciinema.org/a/230671.svg)](https://asciinema.org/a/230671?autoplay=1)
 
-## Pre-requirements
 
-- Python 3.8 or greater.
-- The Python3 binding for libvirt, the package is probably called `python3-libvirt`.
-- You can also just build the binding during the install, in this case, you will need to install some packages first:
-    - `dnf install libvirt-devel python3-devel` (Fedora/RHEL)
-    - `apt install libvirt-dev python3-dev` (Debian/Ubuntu)
-- You make also want to install `python3-urwid` if you want to get the fancy list of VM. This dependency is optional.
+## Requirements
+- 1Gb storage space
+- 100Mb ram
```

Comment:
I feel bad for the person who will try this RAM and storage combination :-).

Reply:
But Bill Gates said i'ld never need more than 640Kb of ram.....

Bloat is one of my pet peeves, I was not accounting for VM's  ect. I guess thou these requirements arn't really common any more.